webapp/src/Command/ScoreboardMergeCommand.php

```diff
@@ -307,7 +308,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         // Render the scoreboard to HTML and print it.
         $data = $this->scoreboardService->getScoreboardTwigData(
-            null, null, '', false, true, true, $contest, $scoreboard
+            null,
+            null,
+            '',
+            false,
+            true,
+            true,
+            $contest,
+            $scoreboard
```

Comment:
Why not simply put everything on a single line? This wastes a lot of lines with little content.

Reply:
I agree that there is no benefit in splitting this over multiple lines. Also when looking at https://www.php-fig.org/psr/psr-2/#46-method-and-function-calls the splitting is a `MAY` not a `MUST`.